src/components/MainPage/MainPage.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import './MainPage.scss';
+
+import { user } from '../../redux/gateway';
+
+const MainPage = () => (
+  <div className="main__page">
+    <h2 className="main__page_header">
+      Hello,<span className="main__page_header-span">{` ${user}!`}</span>
+    </h2>
+    <p className="main__page_greeting">What would you like to do?</p>
+    <div className="main__page_buttons">
+
+      <button className="main__page_buttons-btn btn">
```

Comment:
OK, if you have the design as you have I can suggest creating a separate component `NavLink` with props text and link and move there styles ;)

Reply:
Fixed. Separate NavLink component created and used over the app =)